arch/arm/src/stm32/stm32_comp.h

```diff
@@ -192,5 +84,4 @@ FAR struct comp_dev_s *stm32_compinitialize(int intf);
 #endif
 #endif /* __ASSEMBLY__ */
 
-#endif /* CONFIG_STM23_COMP */
-#endif /* __ARCH_ARM_SRC_STM32_STM32_COMP_H */
+#endif /* __ARCH_ARM_SRC_STM32_STM32_COMP_H */
```

Comment:
Recommend adding newline at end of file.

Reply:
> Recommend adding newline at end of file.

I think it is not necessary, just look others header .h files in the same directory and at include/

Reply:
Thanks for helping the review. Done

Reply:
Should I revert the commit 60c5ec0?

Reply:
Don't need to